plugins/chat/assets/stylesheets/common/core-extensions.scss

```diff
@@ -11,6 +11,19 @@
   }
 }
 
+.user-summary-page .details .controls ul {
+  display: flex;
+  flex-direction: column;
+
+  li:first-child {
+    order: -2;
+  }
+
+  li.user-profile-controls-outlet.chat-button {
+    order: -1;
+  }
```

Comment:
Im generally kinda against this, its makes tabbing weird

Reply:
Yeah I don't love that either. Visually it looks good but it potentially introduces usability issues. I think there is still no way to set tab index via css so this is a bit tricky.

There are a few options here:

1) keep as it is but add tabindex="-1" to skip this button when tabbing.

2) leave the Chat button in it's default place when added via plugin outlet, and conditionally change the `btn-secondary` class from the component. So it would look like:

<img width="174" alt="Screenshot 2024-03-14 at 3 57 00 PM" src="https://github.com/discourse/discourse/assets/2257978/f0599fa8-bb98-4f9c-ac1a-533e121a3240">

3) keep this as it currently is but accept that the tabbing isn't in the correct order 

Maybe good to get input from @chapoi here too

Reply:
Going to go with option 3 for now but we can refactor this later if it causes any concern.